contents/privacy.md

```diff
@@ -151,7 +151,7 @@ Additional rights that may apply to you in certain instances:
 
```

Comment:
Flagging these changes as they may have legal or regulatory requirements to update.